app/views/shared/_attributes.html.erb

```diff
@@ -1,8 +1,11 @@
-      <table class="table">
+<% include_title ||= false # title is part of the exterior template in common cases %>
```

Comment:
Looks like this is superceded in the (ostensibly unrelated) PR #1947.  I support the approach there since it does what I projected was necessary.